src/commonComponents/Loader/Loader.js

```diff
@@ -0,0 +1,19 @@
+//Libraries
+import React from 'react';
+
+//Components
+import FadeLoader from 'react-spinners/FadeLoader';
```

Comment:
Right now we should not rely much on npm packages for this purpose!!

Try to write simple loader using vanilla css